sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -399,6 +399,8 @@ Returns an `annotated_ref` reference wrapper which can be used to read or write
 to the underlying pointer. Reads/Writes using the reference will retain the
 annotations.
 
+Only applicable properties will be passed to `annotated_ref`.
```

Comment:
From your changes to sycl_ext_intel_fpga_kernel_arg_properties it seems that you want the default to be that a property is not passed on to annotated_ref  I think you need to state that default here explicitly.  I.e:

```suggestion
Only applicable properties will be passed to `annotated_ref`.  Unless a property's description states otherwise it will not be passed on.  
```